docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<ol><li>nit: <code>Parameters</code> -&gt; <code>Parameter</code>s since the class name is singular.</li><li>The description of observables may be misread. I would suggest something like "One or more observables, each of which can have multiple terms, arranged into an array. A single observable is a 0-d array, a list of observables is a 1-d array (ie vector), etc. Each observable can be specified in any of several formats, such as ...</li><li>suggestion "A collection of parameter values to bind the circuit against θ_k (only needed if any <code>Parameters</code> are used that must be bound at runtime) " -&gt; "A collection of parameter sets to bind the circuit against. This can be specified as a single array-like where the last index is over circuit <code>Parameter</code>s, or omitted (equivalently, set to None) if the circuit has no <code>Parameter</code>s). "</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>